src/main/java/appeng/client/render/NetworkVisualiserRender.java

```diff
@@ -0,0 +1,344 @@
+/*
+ * Copyright (c) bdew, 2014 - 2015 https://github.com/bdew/ae2stuff This mod is distributed under the terms of the
+ * Minecraft Mod Public License 1.0, or MMPL. Please check the contents of the license located in
+ * http://bdew.net/minecraft-mod-public-license/
+ */
```

Comment:
I don't think this is necessary

Reply:
I would update the credits section in the readme to include ae2stuff rather than a comment like this in one class